src/components/GeneralStepper/GeneralStepper.tsx

```diff
@@ -13,6 +13,7 @@ export interface StepperProps {
   defaultBackPathLabel?: string;
   completeBtnText?: string; // Text for btn to navigate out of flow in last step.
   wrapperClassName?: string;
+  basic?: boolean;
```

Comment:
The prop name is ambiguous. eg. What is a non-basic `GeneralStepper`?
Can we choose a more declarative name or comment please